cmd_tools/create_hdusd_libs.py

```diff
@@ -82,11 +82,11 @@ def main():
 
     # Add the arguments to the parser
     ap.add_argument("-usd", required=True, metavar="",
-                    help="USD dir")
+                    help="USD build dir")
     ap.add_argument("-hdrpr", required=True, metavar="",
-                    help="HdRPR dir")
+                    help="HdRPR install dir")
     ap.add_argument("-mx", required=True, metavar="",
-                    help="MaterialX dir")
+                    help="MaterialX build dir")
```

Comment:
same comment here.  I think these should be more explicit if it's the source or built dir